File: app/src/main/java/org/nervos/neuron/util/JSLoadUtils.java
package org.nervos.neuron.util;

import android.webkit.WebView;

/**
 * Created by BaojunCZ on 2018/10/18.
 */
public class JSLoadUtils {
    public static void loadFunc(WebView webView, final String func) {
        webView.loadUrl("javascript:" + func + "()");
    }

    public static void loadFunc(WebView webView, final String func, final String arg1) {
        webView.loadUrl("javascript:" + func + "('" + arg1 + "')");
    }

    public static void loadFunc(WebView webView, final String func
            , final String arg1, final String arg2) {
        webView.loadUrl("javascript:" + func + "('" + arg1 + "','" + arg2 + "')");
    }
}

File: app/src/main/java/com/cryptape/cita_wallet/util/JSLoadUtils.kt
package com.cryptape.cita_wallet.util

import android.webkit.WebView

/**
 * Created by BaojunCZ on 2018/10/18.
 */
object JSLoadUtils {
    fun loadFunc(webView: WebView, func: String) {
        webView.loadUrl("javascript:$func()")
    }

    fun loadFunc(webView: WebView, func: String, arg1: String) {
        webView.loadUrl("javascript:$func('$arg1')")
    }

    fun loadFunc(webView: WebView, func: String, arg1: String, arg2: String) {
        webView.loadUrl("javascript:$func('$arg1','$arg2')")
    }
}
